subjects/java/checkpoints/distinct-substring-lenght/README.md

```diff
@@ -0,0 +1,45 @@
+## Distinct Substring length
+
+### Instructions
+
+Create a class `DistinctSubstringLength` that provides a method to find the length of the longest substring without repeating characters in a given string.
```

Comment:
I would add a definition of "substring": I only understood this from one of the tests.

For example:
>... a substring is a contiguous sequence of [characters](https://en.wikipedia.org/wiki/Character_(computing)) within a [string](https://en.wikipedia.org/wiki/String_(computer_science)).

From [Wikipedia](https://en.wikipedia.org/wiki/Substring)

Reply:
i do it in purpose, not all the informations should be clearly defined in the subject the have to get the informations from the example too.